lifter/PathSolver.cpp

```diff
@@ -779,8 +779,9 @@ PATH_info solvePath(Function* function, uintptr_t& dest,
         value_with_least_possible_values->print(outs());
         outs() << "\n";
         outs().flush();
-        outs() << " bitsof_least_possible_value : "
-               << bitsof_least_possible_value << "\n";
```

Comment:
This wouldn't need to change after we overload << operand

Reply:
Remove ;)